Libraries/Components/Pressable/Pressable.js

```diff
@@ -194,9 +194,9 @@ function Pressable(props: Props, forwardedRef): React.Node {
     ...restProps,
     ...android_rippleConfig?.viewProps,
     acceptsFirstMouse: acceptsFirstMouse !== false && !disabled, // [TODO(macOS GH#774)
-    enableFocusRing: enableFocusRing !== false && !disabled, // ]TODO(macOS GH#774)
+    enableFocusRing: enableFocusRing !== false && !disabled,
     accessible: accessible !== false,
-    focusable: focusable !== false,
+    focusable: focusable !== false && !disabled, // ]TODO(macOS GH#774)
```

Comment:
Why the removal of one TODO tag and the addition of the other? Does upstream even have `enableFocusRing` as a prop?

Reply:
Some of these TODO tags have left or right square brackets, indicating a block of code they are starting or ending, respectively.  Since I'm making a change to this line, I'm moving the end of the TODO block down to this line so that this change is also covered by the block.

Reply:
Alternatively I could leave the existing end mark where it is and add a single-line comment to the `focusable` line, but as written seemed simpler.